Makefile

```diff
@@ -7,23 +7,11 @@ models: \
 	drafttopic_models
 
 drafttopic_models: \
-	models/arwiki.drafttopic.gradient_boosting.model \
-	models/cswiki.drafttopic.gradient_boosting.model \
-	models/enwiki.drafttopic.gradient_boosting.model \
-	models/kowiki.drafttopic.gradient_boosting.model \
-	models/viwiki.drafttopic.gradient_boosting.model
+	models/ukwiki.drafttopic.gradient_boosting.model

```

Comment:
You'll need to add the other models back.

Reply:
Yes. The current Makefile is only for building purposes. I'll restore.